### PR TITLE
[transactions] Implement transactions details page

### LINKED ISSUE #1263: [Transactions] Implement transactions details page

### PR DESCRIPTION
**Pull request by jorgeepc**
This PR implements the transactions detail page.

## Changes

- New `Transaction` detail page
- New `PaginatedList` component
- Refactored the 'Test` detail page

## Fixes

- fixes #1263 

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: web/src/components/RunCard/RunCard.tsx [hunk 1]
```diff
@@ -24,17 +25,18 @@ function getIcon(state: TTestRun['state'], failedAssertions: number) {
   return <S.IconSuccess />;
 }
 
-const ResultCard = ({
+const RunCard = ({
```
Comment (xoscar) [inline web/src/components/RunCard/RunCard.tsx:28]: Are we reusing this card for the moment or do you think we'll keep the same one after we integrate the BE services?

File: web/src/components/TestHeader/TestHeader.tsx [hunk 2]
```diff
@@ -1,35 +1,28 @@
...
-        </div>
-      </S.Section>
-      <S.Section>
-        <TestCardActions testId={id} onDelete={() => onDelete(test)} />
-      </S.Section>
-    </S.Container>
-  );
-};
+const TestHeader = ({description, id, onBack, onDelete, title}: IProps) => (
+  <S.Container $isWhite={!ExperimentalFeature.isEnabled('transactions')}>
```
Comment (xoscar) [inline web/src/components/TestHeader/TestHeader.tsx:14]: White privileges

File: web/src/models/Transaction.model.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import {TRawTransaction, TTransaction} from 'types/Transaction.types';
+
+const Transaction = ({id = '', name = '', description = '', version = 1}: TRawTransaction): TTransaction => ({
```
Comment (xoscar) [inline web/src/models/Transaction.model.ts:3]: Dude check what @cescoferraro did here https://github.com/kubeshop/tracetest/pull/1320 for the model as well, so we don't have issues moving forward

File: web/src/pages/Environments/EnvironmentCard.tsx [hunk 4]
```diff
@@ -42,15 +42,15 @@ export const EnvironmentCard = ({
     <E.EnvironmentCard $isCollapsed={isCollapsed}>
       <E.InfoContainer onClick={toggleColapsed}>
         {isCollapsed ? <DownOutlined /> : <RightOutlined data-cy={`collapse-environment-${id}`} />}
-        <T.TextContainer>
+        <E.TextContainer>
           <E.NameText>{name}</E.NameText>
-        </T.TextContainer>
-        <T.TextContainer />
-        <T.TextContainer data-cy={`environment-description-${id}`}>
+        </E.TextContainer>
```
Comment (xoscar) [inline web/src/pages/Environments/EnvironmentCard.tsx:47]: E? 🤔 😂

File: web/src/pages/Test/Content.tsx [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+          test.trigger.entryPoint
+        }`}
+        id={test.id}
+        onBack={() => navigate('/')}
+        onDelete={() => onDelete(test)}
+        title={`${test.name} (v${test.version})`}
+      />
+
+      <S.ActionsContainer>
+        <div />
```
Comment (xoscar) [inline web/src/pages/Test/Content.tsx:42]: Is this div required?

File: web/src/pages/Test/Content.tsx [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+          ghost
+          loading={isLoadingRunTest}
+          onClick={handleRunTest}
+          type="primary"
+        >
+          Run Test
+        </Button>
+      </S.ActionsContainer>
+
+      <PaginatedList<TTestRun, {testId: string}>
```
Comment (xoscar) [inline web/src/pages/Test/Content.tsx:55]: typed component 🤓 🪄

**Review by xoscar — APPROVED** [review]
GJ @jorgeepc I left some non-blocking comments and questions!

File: web/src/components/TestHeader/TestHeader.tsx (hunk 2 above)
Comment (cescoferraro) [reply web/src/components/TestHeader/TestHeader.tsx:14]: 🤣

**Pull request merged**